src/main/java/de/bonndan/nivio/input/ProcessLog.java

```diff
@@ -91,6 +91,15 @@ public String getError() {
         return exception.getMessage();
     }
 
+    @JsonIgnore
+    public Date getLastUpdate() {
```

Comment:
I would advise you not to use ``Date``, if you have a choice.
The new, immutable date types like ``ZonedDateTime``  or ``LocalDateTime`` are much better.

Reply:
Fixed in https://github.com/dedica-team/nivio/commit/348b9bf2ae3ec6c0acb374c8ccb3b31d188dc129